processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/TableLoaderGenerator.java

```diff
@@ -436,6 +447,11 @@ private MethodSpec generateLoadMissingFileMethod() {
                 classNames.tableContainerTypeName(),
                 classNames.tableContainerTypeName(),
                 TableStatus.class)
+            .beginControlFlow("if (isRecommended())")
+            .addStatement(
+                "noticeContainer.addValidationNotice(new $T(gtfsFilename()))",
+                MissingRecommendedFileNotice.class)
+            .endControlFlow()
```

Comment:
I think this could be added in the other PR, but everything else in this class could stay.